ga4gh/frontend.py

```diff
@@ -228,6 +274,9 @@ def configure(configFile=None, baseConfig="ProductionConfig", extraConfig={}):
     theBackend.setDefaultPageSize(app.config["DEFAULT_PAGE_SIZE"])
     theBackend.setMaxResponseLength(app.config["MAX_RESPONSE_LENGTH"])
     app.backend = theBackend
+    app.secret_key = os.urandom(SECRET_KEY_LENGTH)
```

Comment:
We need a comment here and a follow up issue --- how do we set this (secret_key) up in general? What happens for Apache and Docker deployments?